packages/container/lib/registry.js

```diff
@@ -18,23 +18,34 @@ var VALID_FULL_NAME_REGEXP = /^[^:]+.+:[^:]+$/;
  @class Registry
 */
 function Registry(options) {
+  this.failover = options && options.failover ? options.failover : null;
```

Comment:
This is a total bikeshed, but I'm not _totally_ in love with the term "failover." To me, a failover implies latch-like behavior: once the "fail" has occurred, you switch everything over to the failover. For example, if the power goes out, you have a "failover" generator that everything switches to.

In this case, it's not really a failover per se… it just looks in its local one first, then expands its search to the parent registry if it's available. Perhaps this terminology is controversial, but I think I'd almost prefer `parentRegistry` to `failover`.